10up/distributor
# Fix/1144 webp support

### Description of the Change


This PR:

- [x] adds WebP support to Distributor, for parity with WordPress 5.8
- [x] adds HEIC support to Distributor, for parity with WordPress 6.5
- [x] adds AVIF support to Distributor, for parity with WordPress 6.7
- [ ] adds tests:
    - [ ] WebP
    - [ ] HEIC
    - [ ] AVIF


Closes #810

### How to test the Change


Upload test AVIF, HEIC, and WebP files to a Distributor source site. Ensure taht they are synchronized to a downstream Distributor site.

### Changelog Entry


> Added - Support for WebP, paralleling WordPress 5.8
> Added - Support for HEIC, paralleling WordPress 6.5
> Added - Support for AVIF, paralleling WordPress 6.7

### Credits


~~Props @username, @username2, ...~~

### Checklist:



    - [ ] existing
    - [ ] new
